Vector.h

```diff
@@ -295,7 +295,7 @@ namespace NCL::Maths {
         template <typename T, uint32_t n>
         constexpr T		GetMinElement(const VectorTemplate<T, n>& a) {
             float v = a.array[0];
-            for (int i = 1; i < n; ++i) {
+            for (int i = 0; i < n; ++i) {
```

Comment:
This change is incorrect - We can start the loop at 1, as v already has the value of the array at 0, so there's no point comparing.